lib/koala/api/graph_batch_api.rb

```diff
@@ -18,7 +18,7 @@ def batch_calls
         @batch_calls ||= []
       end
 
-      def graph_call_in_batch(path, args = {}, verb = "get", options = {}, &post_processing)
+      def graph_call_in_batch(path, args = {}, verb = 'get', options = {}, &post_processing)
```

Comment:
Good catch! I personally prefer double quotes but the usage should be consistent with the file.